### DISCUSSION TITLE
Welcome to Weblate Discussions!

### DISCUSSION THREAD

**Discussion by nijel** [WeblateOrg/weblate#4691] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilocit** [comment]
Seems to be a good solution. 
Especially the advantage of having most of everything in one place (GitHub).
Thanks, @nijel

**Reply by comradekingu** [reply]
nb_NO translator. I try to help where I can, and have some ideas for what a future translation platform could be.

**Comment by yarons** [comment]
Hi there, nice to see you here also :)

**Comment by slsimic** [comment]
sr_RS translator of Weblate and some other things...
I must say, every time I return to hosted.weblate.org I get more frustrated. :-)
Every time I see more and more people joining to participate but also significantly dropping the quality of the translations.
I really think that hierarchy of users must exist and project maintainers must be warned and aware that almost always quantity does not mean quality.
Many people came to translate few strings just to be able to call themselfs a "translator" disregarding basic rules of their own languages and disregarding previous translations, altough they can see in "Nearby strings" how it's suppose to be.

So, project managers, coordinators, reviewers, proofreaders... call it as you want... need to exist in order you to have a good translations. Hell, even translator title should be earned.

**Reply by ilocit** [reply]
@slsimic  Slobodan, this is a very valid point. I guess you have some experience with "real" translations. I would also be interested in developing such a hierarchical user concept. Guess we need to find one or two more to make this a topic which we can drive.

**Reply by nautilusx** [reply]
@nijel I can fully support the proposal of @slsimic . We absolutely need this solution to maintain and improve the translation quality. I have the same experience with the German translations. We need project managers, coordinators and reviewers.

**Reply by nijel** [reply]
The reviewing is already possible, you just need to turn it on, see https://docs.weblate.org/en/latest/workflows.html

**Reply by nijel** [reply]
PS: If you want to discuss this further, I suggest starting a new discussion.

**Reply by comradekingu** [reply]
@slsimic I find if you point out said rules in the comments bad translators stop their transgressions.
You don't actually need a hierarchy of roles, and having one isn't a solution. If per-user reviews were a thing it would be possible to share the only concept of quality that is ultimately valid. https://github.com/WeblateOrg/weblate/issues/2699
